SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -156,7 +156,7 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	uint32 MaxDynamicallyAttachedSubobjectsPerClass;
 
 	/** EXPERIMENTAL - This is a stop-gap until we can better define server interest on system entities.
-	Disabling this is not supported in any type of multi-server environment*/
+	Disabling this is not supported in a zoned-worker environment*/
```

Comment:
Does that mean it's fine for Offloading?

Reply:
Yes